utils/build/docker/nodejs/express4.Dockerfile

```
@@ -7,13 +7,14 @@ RUN uname -r
 # print versions
 RUN node --version && npm --version && curl --version
 
-COPY utils/build/docker/nodejs/express4 /usr/app
+COPY utils/build/docker/nodejs/express /usr/app
 
 WORKDIR /usr/app
 
 ENV NODE_ENV=production
 
 RUN npm install
+RUN npm install "express@^4.17.2" "apollo-server-express@^3.13.0" "express-mongo-sanitize@^2.2.0"
```

as package-lock.json will not work for these dependencies, should we fix the versions?

```suggestion
RUN npm install "express@4.17.2" "apollo-server-express@3.13.0" "express-mongo-sanitize@2.2.0"
```

---

Done